src/js/components/TodoList.js

```diff
@@ -0,0 +1,54 @@
+import { filter } from "../lib/renderingFilter.js";
+
+class TodoList {
+  constructor({ memberIndex, filterType }) {
+    this.memberIndex = memberIndex;
+    this.filterType = filterType;
+  }
+
+  render = (items) => {
+    if (!items) return (items = []);
+
```

Comment:
```suggestion
  render = (items=[]) => {
```
이렇게 기본값으로 빈 배열을 받으면 아래의 코드가 동작하지 않으니 원하는 대로 동작할 것 같습니다!

Reply:
좋은 방법이네요 !